packages/flutter/test/gestures/events_test.dart

```diff
@@ -12,7 +12,6 @@ void main() {
 
   testGesture('toString control tests', (GestureTester tester) {
     expect(const PointerDownEvent(), hasOneLineDescription);
-    expect(const PointerDownEvent().toStringFull(), hasOneLineDescription);
```

Comment:
Can you add a test back that verifies that the new toString output is something sensible after this change?

Reply:
Isn't that what the line above is doing? The description of that expectation is "Asserts that an object's toString() is a plausible one-line description."

Reply:
You're right. I totally missed that line...